### DISCUSSION TITLE
cmake_minimum_required

### DISCUSSION THREAD

**Discussion by koen-n** [offa/influxdb-cxx#97] [Q&A] [answered]
cmake_minimum_required was set to 3.12.0 ( #27 ). I'm wondering why 3.12 is set as minimum, as the project can no longer be built on Ubuntu 18.04 with the latest cmake version [from the package archive](https://packages.ubuntu.com/bionic/cmake).

**Comment by offa** [comment, marked as answer]
There was some special handling of versions < 3.12, but no reason not to update.

But Ubuntu Bionic having 3.10 is a good reason to lower the requirements – thanks for reporting, I'll create an issue to address this.

**Reply by koen-n** [reply]
Thanks for the quick response
